dev/start

```diff
@@ -129,6 +133,21 @@ if [[ $ENABLE_AUTHN_LDAP = true ]]; then
   docker-compose exec conjur conjurctl policy load cucumber /src/conjur-server/dev/files/authn-ldap/policy.yml
 fi
 
+if [[ $ENABLE_AUTHN_AZURE = true ]]; then
+  enabled_authenticators="$enabled_authenticators,authn-azure/prod"
+
+  . ../ci/authn-azure/check_dependencies.sh
```

Comment:
We are checking this twice; once here and once in cli. Do we need these both?

Reply:
yes because these are 2 scripts that need the variables.